backend/control/control.go

```diff
@@ -34,130 +37,78 @@ type CartographyJobRequest struct {
 	DefaultRegion      string `json:"default_region,omitempty"`
 }
 
-// RuleExecutionLoop tries to execute rules when
-// 1) The ExecuteRules boolean is true
-// 2) The cartography job is not running.
-// When these conditions are true, it also marks ExecuteRules false.
-// And it sets the LastScanCompleted field - right now, for *all* accounts
-func RuleExecutionLoop(postgresDb *gorm.DB, driver neo4j.Driver, ruleDataList []models.RuleData,
-	rulesToExecute []types.Rule) {
+func ExecuteRules(postgresDb *gorm.DB, driver neo4j.Driver, ruleDataList []models.RuleData,
```

Comment:
Add a comment that explains the two things this function does - ie try executing the rules and update to the postgres db